.github/workflows/tests_check.yml

```diff
@@ -0,0 +1,26 @@
+name: RxPlayer Tests Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - tests/**
+
+jobs:
+  scripts_lint:
```

Comment:
it's named "scripts_lint" because it's a script that lint or because it lint the "script" folder ?
here it's the test folder so it shoud not be named this way :)

Reply:
Yeah I forgot updating the job name here!

Reply:
Fixed that typo